media/src/project/project-map-sidebar.tsx

```diff
@@ -2,15 +2,114 @@ import React, { useState } from 'react';
+        setEventName(e.target.value);
+    };
+
+    const handleDescription= (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+        setDescription(e.target.value);
+    };
+
+    const handleDatetime = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setDatetime(e.target.value);
+    };
+
+    const handleFormSubmbit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        updateEvent(
+            eventName, description, activeEventEdit.location.lng_lat[1], activeEventEdit.location.lng_lat[0], activeEventEdit.pk, activeEventEdit.layer);
+        setActiveEventEdit(null);
+    };
+
+    const handleCancel = (e: React.MouseEvent) => {
+        setActiveEventEdit(null);
+    };
+    return (
+        <>
+            <h4>Edit Event</h4>
+            <form onSubmit={handleFormSubmbit} >
+                <div className={'form-row'}>
+                    <div className={'form-group col-3'}>
+                        <label htmlFor={'event-form__name'}>Name</label>
+                    </div>
+                    <div className={'form-group col-9'}>
+                        <input
+                            type={'text'}
+                            id={'event-form__name'}
+                            value={eventName}
+                            placeholder={'Untitled Marker'}
+                            onChange={handleName}/>
+                    </div>
+                </div>
+                <div className="form-row">
+                    <div className={'form-group col-3'}>
+                        <label htmlFor={'event-form__description'}>
+                            Description
+                        </label>
+                    </div>
+                    <div className={'form-group col-9'}>
+                        <textarea
+                            id={'event-form__description'}
+                            value={description}
+                            rows={3}
+                            onChange={handleDescription}>
+                        </textarea>
+                    </div>
+                </div>
+                <div className="form-row">
+                    <div className={'form-group col-3'}>
+                        <label htmlFor={'event-form__date'}>
+                            Date
+                        </label>
+                    </div>
+                    <div className={'form-group col-9'}>
+                        <input
+                            type={'datetime-local'}
+                            id={'event-form__date'}
+                            value={datetime}
+                            onChange={handleDatetime}/>
+                    </div>
+                </div>
+                <div className="form-row">
+                    <div className={'form-group col-3'}>
+                    </div>
+                    <div className={'form-group col-9'}>
+                        <button onClick={handleCancel} className={'btn btn-danger'}>
```

Comment:
When I click "enter" in the Add or Edit marker form, the default button seems to be "Cancel", which was surprising. I would assume "enter" would submit and "esc" would cancel.

Reply:
Fixed